components/places/src/storage/history.rs

```diff
@@ -42,12 +41,10 @@ static DELETION_HIGH_WATER_MARK_META_KEY: &str = "history_deleted_hwm";
 
 /// Returns the RowId of a new visit in moz_historyvisits, or None if no new visit was added.
 pub fn apply_observation(db: &PlacesDb, visit_ob: VisitObservation) -> Result<Option<RowId>> {
-    breadcrumb!("apply_observation: begin_transaction");
```

Comment:
Could this be useful in the future? I'm not sure, but would it be possible breadcrumbing here can help us track other issue?

Reply:
Maybe useful, but also maybe noise.  I'm not sure what the best approach is to the breadcrumbs, hopefully we can figure it out as we go along.